python/attack_vid_labels.py

```diff
@@ -0,0 +1,350 @@
+AV_LABELS = {
+    "CNAU0235000H_exact_match.mp4" : "CNAU0235000H.mpg",
```

Comment:
is this generated from a script, if not it might be more maintainable if this was generated

Reply:
Making this an issue to change later. I have to think about how robust to make the script.

#13